homeassistant/components/litterrobot/const.py

```diff
@@ -1,2 +1,7 @@
 """Constants for the Litter-Robot integration."""
+from typing import Union
+
+from pylitterbot import FeederRobot, LitterRobot
+
 DOMAIN = "litterrobot"
+RobotTypes = Union[FeederRobot, LitterRobot]
```

Comment:
Would a Generic work? If it does it would be better.

Reply:
Note: I don't think you understood what I meant about using Generics.
I have created a follow-up PR here: #77537

Reply:
Thanks, that makes more sense now.